config/secrets.yml

```diff
@@ -17,17 +17,29 @@
 
 # Environmental secrets are only available for that specific environment.
 
-development:
-  secret_key_base: 1b7333657709a8e14d929c518b4befc85655aa71c0f9bd09250123f1d5b2273c0b21c4399e9a68c8b72c356b48475ac856efce63deb08781505a691f76a95aec
+common: &common
   username: <%= ENV["USERNAME"] %>
   password: <%= ENV["PASSWORD"] %>
   address: <%= ENV["ADDRESS"] %>
   port: <%= ENV["PORT"] %>
   domain: <%= ENV["DOMAIN"] %>
   authentication: <%= ENV["AUTHENTICATION"] %>
+  from@email: <%= ENV["FROM@MAIL"] %>
+
+development:
+  secret_key_base: 1b7333657709a8e14d929c518b4befc85655aa71c0f9bd09250123f1d5b2273c0b21c4399e9a68c8b72c356b48475ac856efce63deb08781505a691f76a95aec
+  <<: *common
 
 test:
   secret_key_base: 07822b15f8f8e1b6fee95d6fc60e31167b017e67f26f6929bcbdc396a073a635b0c113d399e55970e1ef0c1aa973369c4f46a5cdf640ab0627107e59244ed866
+  from@email: "agustin.bajo@wolox.com.ar"
+  username: 'c40b284bf243d4'
+  password: '786524a3382560'
+  address: 'smtp.mailtrap.io'
+  port: '2525'
+  domain: 'smtp.mailtrap.io'
+  authentication: :cram_md5
```

Comment:
this should be using values from the .env file like in the common section in this file.